sshd/issue_invite.sh

```diff
@@ -28,36 +28,37 @@ done
 shift $((OPTIND-1))
 
 TMP=`mktemp -d`
+npm install jsurl &>/dev/null
```

Comment:
Ah...yeah, I dunno why that won't work but I get the same error after running `npm install -g jsurl` on my system. Weird! Let's worry about this later.